Tools/Scripts/libraries/webkitcorepy/webkitcorepy/subprocess_utils.py

```diff
@@ -44,12 +45,20 @@ def run(*popenargs, **kwargs):
 
         if difference:
             timeout = min(timeout or sys.maxsize, int(math.ceil(difference)))
-        if capture_output:
+        if capture_output or capture_and_stream_output:
             if ('stdout' in kwargs) or ('stderr' in kwargs):
-                raise ValueError('stdout and stderr arguments may not be used with capture_output.')
+                raise ValueError('stdout and stderr arguments may not be used with capture_output and capture_and_stream_output.')
             kwargs['stdout'] = subprocess.PIPE
-            kwargs['stderr'] = subprocess.PIPE
-        return subprocess.run(*popenargs, timeout=timeout, **kwargs)
+            kwargs['stderr'] = subprocess.STDOUT
```

Comment:
This is a massive change of semantics that we almost certainly cannot make without breaking existing code, and moves us away from having something API-compatible with `subprocess.run`.